control-plane/agents/src/bin/core/tests/controller/mod.rs

```diff
@@ -81,6 +81,7 @@ async fn store_lease_lock() {
         .await
         .unwrap();
 
+    tokio::time::sleep(std::time::Duration::from_secs(1)).await;
```

Comment:
Do we need this sleep?

Reply:
Seems on gha etcd is not ready without it, not sure why atm...
I'll add a comment for the effect